cosmo_tester/test_suites/test_blueprints/managed_plugins_test.py

```diff
@@ -0,0 +1,97 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import os
+import tarfile
+import filecmp
+import tempfile
+
+from cosmo_tester.framework.testenv import TestCase
+
+from wagon.wagon import Wagon
+
+TEST_PACKAGE_NAME = 'django'
+TEST_PACKAGE_VERSION = '1.6'
+
+
+class DownloadInstallPluginTest(TestCase):
+
+    def setUp(self):
+        super(DownloadInstallPluginTest, self).setUp()
+        self.wheel_tar = self._create_wheel(TEST_PACKAGE_NAME,
+                                            TEST_PACKAGE_VERSION)
+        self.downloaded_archive_path = os.path.join(self.cfy.workdir,
+                                                    os.path.basename(
+                                                        self.wheel_tar))
+
+    def tearDown(self):
```

Comment:
no need for any `tearDown` method, as the base class takes care of removing the entire `self.workdir` after each test